### PR TITLE
fix css and don't disable input lable

### PR DESCRIPTION
**Pull request by sudhirverma**
Screenshot 2022-04-01 at 11 12 43 PM
Screenshot 2022-04-01 at 11 12 56 PM

### CODE REVIEW

**Comment by robstryker** [comment]
I don't agree with @mohitsuman  on the ellipsis for "Browse". 

https://docs.microsoft.com/en-us/windows/win32/uxguide/ctrl-command-buttons?redirectedfrom=MSDN

image

Most style guides say that an ellipsis should be used when a button 1) opens a new window, and 2) that window requries additional information from a user.  I believe both eclipse and intellij insisted on this style. I haven't found anything in the vscode style guide (or even found the guide, to be honest) but the browse button should have ellipsis as far as I am concerned. 

I will merge the other portion of the PR.

**Comment by robstryker** [comment]
image
https://jetbrains.design/intellij/controls/button/

**Comment by robstryker** [comment]
As to your second issue, style guides seem to indicate a label should be disabled too if a field is disabled. 

https://jetbrains.design/intellij/controls/input_field/
image

**Comment by mohitsuman** [comment]
The image reflected above does not show the full context. The text also mentions that;
` This doesn't mean you should use an ellipsis whenever an action displays another window only when additional information is required to perform the action. Consequently, any command button whose implicit verb is to "show another window" doesn't take an ellipsis` (reference from Microsoft documentation)

Given the design pattern of latest UI frameworks such as Bootstrap, Patternfly(based on bootstrap and official from Red Hat), the buttons do not have ellipsis and have text labels as each serves its own semantic purpose.

And with the vscode theme, I want it to be inclined as much as with possible with bootstrap/patternfly . This is how it works in bootstrap, the `Select File` button has no ellipsis https://getbootstrap.com/docs/4.0/components/forms/

**Comment by mohitsuman** [comment]
> As to your second issue, style guides seem to indicate a label should be disabled too if a field is disabled.
> 
> https://jetbrains.design/intellij/controls/input_field/ image

Yes I think this should be fine, we disable both label and text box together. 👍🏼

**Comment by robstryker** [comment]
> And with the vscode theme, I want it to be inclined as much as with possible with bootstrap/patternfly . This is how it works in bootstrap, the `Select File` button has no ellipsis https://getbootstrap.com/docs/4.0/components/forms/

image

From the same link you just sent me...

**Comment by robstryker** [comment]
This bootstrap style guide seems really contradictory and bipolar. They can't seem to stay consistent. I... immediately put less faith in this guide. I'm include to reject this entire request at this point.

**Comment by mohitsuman** [comment]
Surprisingly I see a different view 

Screenshot 2022-04-02 at 12 57 35 AM

And to refer the latest bootstrap for file selection, this is how they use https://getbootstrap.com/docs/5.1/forms/form-control/#file-input

**Comment by robstryker** [comment]
Yeah this is weird as hell.  Using the 5.1 link (which you just used), I see this:

image

**Comment by mohitsuman** [comment]
I am more inclined to why the views are different in your case. I am using Chrome/Safari on macOS. Are you using some other browser/system ?

**Comment by lstocchi** [comment]
> > As to your second issue, style guides seem to indicate a label should be disabled too if a field is disabled.
> > https://jetbrains.design/intellij/controls/input_field/ image
> 
> Yes I think this should be fine, we disable both label and text box together. 👍🏼

I don't fully agree with this and i think there is a misunderstanding.
The label and field should be disabled if they are part of a section which can be enabled somehow, like 
image

but if you have a field (read-only/disabled) and contains a default value that is shown for reference (which can be updated or not) then the label is always active because that value is used and not ignored at runtime.
image

In the example above even if the textfield is disabled, IJ will use that path to find my settings. Then i'm free to update it or not. 
Opposite if i see that the label is disabled i know that the value will be not used unless i enabled it.

In our use-case we show the path as a reference because the user is trying to invoke a function opened in vscode, and that path is always used so it is a value always "active". The field is read-only though because there is no need for the user to update it, he already selected it from the tree when activating the invoke action.

In conclusion, the textbox needs a readonly flag which will let the label being not disabled or another rule to being able to disable only the textbox

File: src/StandardWizardPageRenderer.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class StandardWizardPageRenderer implements IWizardPageRenderer {
     const htmlSection =
       `<section id="${id}" class="section--settings section--collapsible" >
         <div class="section__header" onclick="document.getElementById('${id}').classList.toggle('collapsed');" >
-          <h2>${label}</h2>
+          <span>${label}</span>
```
Comment (sudhirverma) [inline src/StandardWizardPageRenderer.ts:54]: @robstryker can you look in this PR @mohitsuman want this <h2> tag small.

**Comment by robstryker** [comment]
PR is out of date
